exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,14 @@
+var total = 23;
+
+// what is number of students as a % of the total
 var numberOfStudents = 15;
+
+var percent = (numberOfStudents * 100) / total;
+console.log("Percentage student: " + Math.round(percent) + "%");
+// ------------------------
+
+//what is number of mentors as a % of the total
 var numberOfMentors = 8;
+
+var percent = (numberOfMentors * 100) / total;
```

Comment:
Just a small thing, but try to be more specific with the naming of the `percent` variable here and on line 6 to avoid repeating the same word e.g. `percentageOfMentors`